cli/src/templates/packages/nativewind/components/Button.tsx.ejs

```diff
@@ -1,11 +1,11 @@
 import { forwardRef } from 'react';
-import { Text, TouchableOpacity, TouchableOpacityProps } from 'react-native';
+import { Text, TouchableOpacity, TouchableOpacityProps, View } from 'react-native';
 
 type ButtonProps = {
   title: string;
 } & TouchableOpacityProps;
 
-export const Button = forwardRef<TouchableOpacity, ButtonProps>(({ title, ...touchableProps }, ref) => {
+export const Button = forwardRef<View, ButtonProps>(({ title, ...touchableProps }, ref) => {
```

Comment:
This was erroring because TouchableOpacity's seems to have changed and is no longer a type its just a value (function)

Reply:
if you see the touchable opacity ref type its actually ref<View> so maybe this is actually fine

Reply:
should we be using Pressable in place of TouchableOpacity?

Reply:
Probably yeah